File: executable/r/basic-histogram.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

x = rnorm(500)

data <- list(
  list(
    x = x, 
    type = "histogram"
  )
)

response <- p$plotly(data, kwargs=list(filename="basic-histogram", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/node/basic-histogram.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var x = Math.random() * 500

var data = [
  {
    x: x, 
    type: "histogram"
  }
];

var graph_options = {filename: "basic-histogram", fileopt: "overwrite"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});